Makefile

```diff
@@ -70,6 +70,7 @@ MANAGER_IMAGE_PATCH := "apiVersion: apps/v1\
 \n        - --exempt-namespace=${GATEKEEPER_NAMESPACE}\
 \n        - --operation=webhook\
 \n        - --operation=mutation-webhook\
+\n        - --operation=generate\
```

Comment:
this is added to controller-manager instead of audit. was this intentional?

Reply:
This was supposed to be added to audit pod. Fixed it.